grafana-plugin/e2e-tests/settings/tabs.test.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '../fixtures';
+import { goToOnCallPage } from '../utils/navigation';
+
+test(`tab query param is used to show proper page tab`, async ({ adminRolePage }) => {
+  const { page } = adminRolePage;
+  goToOnCallPage(page, `settings`, { tab: 'ChatOps' });
+
+  const tab = await page.locator("button[aria-label='Tab Chat Ops']");
```

Comment:
1. We don't need await here
2. We should avoid css or XPath selectors as mentioned [here](https://playwright.dev/docs/best-practices#prefer-user-facing-attributes-to-xpath-or-css-selectors). Here's the list of recommended built-in locators: https://playwright.dev/docs/locators#quick-guide 

In this case we can use `page.getByRole('tab', { name: 'Chat Ops' });`

```suggestion
  const tab = page.getByRole('tab', { name: 'Chat Ops' });
```